bitrise.yml

```diff
@@ -12,50 +12,144 @@ workflows:
     - errcheck:
     - go-test:
     after_run:
-    - apk_deploy_test
+    - apk_deploy_test_1
     - aab_deploy_test
 
-  aab_deploy_test:
+  _build_start:
```

Comment:
Do not think _build_start is referenced anywhere.

Reply:
If merged (or just before merge) I want to replace the "after" workflows to run this. The reason I did not do this, is that the workflows are read from the master only, when triggering the build.